### ISSUE TITLE
bad argument type: generator

### ISSUE THREAD

**Issue by mjrk** [arskom/spyne#445]

**Comment by plq** [comment]
What are your requirements?

**Comment by mjrk** [comment]
Sorry, I missed the tornado requirement == 4.2. The `spyne.server.wsgi.WsgiApplication` is served by `tornado.httpserver.HTTPServer` using `tornado.wsgi.WSGIContainer`

All other requirements implicit. What else exactly do you want to know?

**Comment by plq** [comment]
no, I tried to ask you why `Iterable(Unicode)` does not fit your requirements.

**Comment by mjrk** [comment]
I would like to return a primitive type, e.g. `Unicode`.

If this is not allowed for some reason, a more expressive error message could be helpful.

**Comment by plq** [comment]
ah, so you want to return a stream of string?

it's not possible with generators because technically you're returning an array of strings.

Let me see.

**Comment by mjrk** [comment]
Not sure, if this was clear. I want to return 

    @rpc(Unicode, _returns=Unicode)

but I get the above error. To return an array of string (what I do not want) using `Iterable` or `Array` works though.
